### DISCUSSION TITLE
Bottom side LEDs flash randomly

### DISCUSSION THREAD

**Discussion by grannymaster** [awawa-dev/HyperHDR#311] [Q&A]
For some reason, the bottom and some of the  side LEDs flash with white randomly. Usually this happens with dark/darkish picture is displayed, but also happens with bright picture.

I tried cropping the image capture a little, but that didn't help. Couldn't find any other settings which would affect this.
https://i.imgur.com/dAl7az9.png

Logs:
https://pastebin.com/j2DufGHW

**Comment by awawa-dev** [comment]
Do you have a level shifter and that bottom segment is connected just to the date input or it's the end of the LED strip? Assuming it's built correctly using the level shifter and the common ground is provided: since it's not limited to one segment only (but also part of the neighbor so it's rather not the connection problem) and if that happens still in the same area (and the LED strip from the input to this segment is fine) then most likely the LED strip is damaged.

Try to locate first LED (from the input) that is flashing, verify it few times if it starts in the same place and replace that LED with its neighbors. So total 3 LEDs needs to be replaced at least.  But from my experience if the LED strip is beginning to malfunction sooner or later you will have another damaged leds to fix.
